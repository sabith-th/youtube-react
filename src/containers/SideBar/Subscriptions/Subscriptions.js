import React from "react";
import { Divider } from "semantic-ui-react";
import { SideBarHeader } from "../SideBarHeader/SideBarHeader";
import { Subscription } from "./Subscription/Subscription";

export class Subscriptions extends React.Component {
  render() {
    return (
      <React.Fragment>
        <SideBarHeader title="Subscriptions" />
        <Subscription label="Music Channel" broadcasting />
        <Subscription label="Coursera" amountNewVideos={10} />
        <Subscription label="Ted" amountNewVideos={24} />
        <Subscription label="Stanford" amountNewVideos={7} />
        <Subscription label="Udacity" amountNewVideos={116} />
        <Subscription label="MIT" />
        <Divider />
      </React.Fragment>
    );
  }
}
