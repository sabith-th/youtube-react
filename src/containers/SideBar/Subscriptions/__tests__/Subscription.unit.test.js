import { shallow } from "enzyme";
import React from "react";
import { Subscriptions } from "../Subscriptions";

test("renders Subscriptions", () => {
  const wrapper = shallow(<Subscriptions />);
  expect(wrapper).toMatchSnapshot();
});
