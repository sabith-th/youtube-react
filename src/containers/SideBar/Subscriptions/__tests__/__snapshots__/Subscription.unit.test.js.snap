// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Subscriptions 1`] = `
<Fragment>
  <SideBarHeader
    title="Subscriptions"
  />
  <Subscription
    broadcasting={true}
    label="Music Channel"
  />
  <Subscription
    amountNewVideos={10}
    label="Coursera"
  />
  <Subscription
    amountNewVideos={24}
    label="Ted"
  />
  <Subscription
    amountNewVideos={7}
    label="Stanford"
  />
  <Subscription
    amountNewVideos={116}
    label="Udacity"
  />
  <Subscription
    label="MIT"
  />
  <Divider />
</Fragment>
`;
